### PR TITLE
Removing extensions/v1beta1 since deprecated

### PR DESCRIPTION
**Pull request by infa-kparida**
#### Which issue this PR fixes
*(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close that issue when PR gets merged)*
  - fixes #

#### Checklist

- [x] [DCO](https://github.com/prometheus-community/helm-charts/blob/main/CONTRIBUTING.md#sign-off-your-work) signed
- [ ] Chart Version bumped
- [ ] Title of the PR starts with chart name (e.g. `[prometheus-couchdb-exporter]`)

### CODE REVIEW

File: charts/kube-prometheus-stack/Chart.yaml [hunk 1]
```diff
@@ -18,7 +18,7 @@ name: kube-prometheus-stack
 sources:
   - https://github.com/prometheus-community/helm-charts
   - https://github.com/prometheus-operator/kube-prometheus
-version: 12.2.4
+version: 12.2.5
```
Comment (gkarthiks) [inline charts/kube-prometheus-stack/Chart.yaml:21]: Thanks for the PR, please re-base the chart-version and sign the DCO

**Comment by monotek** [comment]
I'm not sure if this is a breaking change and should change major version or if if statement with k8s version compare should be added?

**Comment by scottrigby** [comment]
@monotek that was my first thought, but looking at it, `kubeVersion: ">=1.16.0-0"` was added in the `10.0.0` MAJOR version bump in https://github.com/prometheus-community/helm-charts/pull/128. Unfortunately that dependency wasn't noted in the README "From 9.x to 10.x" section. But in any case, the chart won't install on lower systems so no need to a lower semver compare on this one, right?

**Comment by monotek** [comment]
Ah, did not noticed. So let's go the way it is.

File: charts/kube-prometheus-stack/Chart.yaml (hunk 1 above)
Comment (scottrigby) [reply charts/kube-prometheus-stack/Chart.yaml:21]: DCO is OK now. Chart version does need another bump though, and conflicts resolved.

**Review by monotek — APPROVED** [review]

**Review by vsliouniaev — APPROVED** [review]

**Review by monotek — CHANGES_REQUESTED** [review]
please reabse main / fix merge conflicts.

**Pull request closed without merging**